roles/build_containers/defaults/main.yml

```diff
@@ -55,3 +55,12 @@ cifmw_build_containers_install_from_source: false
 # cifmw_build_containers_dockerfile_labels
 # cifmw_build_containers_extra_config
 # cifmw_build_containers_build_timeout
+
+# Note(Chandankumar): Remove it once watcher related packages are available downstream.
+cifmw_build_containers_exclude_containers:
```

Comment:
cifmw_build_containers_exclude_containers is documented in the README as a list but in fact it's a nested dictionary.

Reply:
Good finding, I will update this pull request to address this change and remove watcher skip from here. thank you!